compiler/src/iree/compiler/Codegen/LLVMCPU/Utils.h

```diff
@@ -52,6 +57,12 @@ FailureOr<Operation *> getRootOperation(ArrayRef<Operation *> computeOps);
 /// aligned.
 bool hasByteAlignedElementTypes(linalg::LinalgOp linalgOp);
 
+/// Sets the tile sizes of the SCFTilingOptions. If `tileScalableFlags` are
+/// provided the corresponding tile size will be multiplied by `vector.vscale`.
```

Comment:
nit: The content in `` usually refers to a variable. I'd just describe it like below

```suggestion
/// provided the corresponding tile size will be multiplied by a vector.vscale op.
```